src/Common/AbstractModel.m

```diff
@@ -132,7 +132,9 @@ function saveObj(obj, suffix)
     end
 
     methods(Access = protected)
+
         function obj = qMRpatch(obj,loadedStruct, version)
+        % This function is to xxx
```

Comment:
You forgot to complete your sentence here I think

Reply:
Oups, I was adding comments here and there and this one slipped through the cracks. Thx for pointing out